src/ztgamemgr.rs

```diff
@@ -0,0 +1,180 @@
+// ztgamemgr module has functions to interact with the live zoo stats such as cash, num animals, species, guests, etc.
+
+use crate::add_to_command_register;
+use crate::debug_dll::{get_from_memory, get_string_from_memory};
+
+use tracing::info;
+use std::collections::HashMap;
+use std::fmt;
+use std::fmt::Display;
+use num_enum::FromPrimitive;
+
+const GLOBAL_ZTGAMEMGR_ADDRESS: u32 = 0x00638048;
+
+// ZTGameMgr struct
+#[derive(Debug)]
+#[repr(C)]
+struct ZTGameMgr {
+    instance: u32, // 0x0
+    get_date: String, // 0x1194 (8 bytes)
+    get_cash: f32, // 0x0C
+    set_cash: f32, // 0x0C
+    add_cash: f32, // 0x0C
+    sub_cash: f32, // 0x0C
+    num_animals: u32, // 0x30
+    num_species: u32, // 0x38
+    num_guests: u32, // 0x54
+    num_tired_guests: u32, // 0x3C
+    num_hungry_guests: u32, // 0x40
+    num_thirst_guests: u32, // 0x44
+    num_guests_restroom_need: u32, // 0x48
+    num_guests_in_filter: u32, // 0x54
+    zoo_admission_cost: f32, // 0x1160
+    enable_dev_mode: bool, // 0x63858A
```

Comment:
I think we already have this somewhere else, in an ini struct

Reply:
Ah, I didn't see them. `enable_dev_mode` probably shouldn't belong in gamemgr anyway, but I wasn't sure what the original structure was. Maybe one of the `UIMgr`s?  The other three are actually part of the game manager structure though. We might want to move the existing hooks or implementations here. Let me know what you think.